### ISSUE TITLE
Sonoff Basic RF and GPIO14 garage door opener and sensor request?

### ISSUE THREAD

**Issue by senadaruc** [robertklep/name.klep.sonoff#22]
Dear Klep,

I have Sonoff Basic RF with Tasmota 6.4.1  configured with "PulseTime - Inching" function to open/close my garage door and with GPIO14 connected into the magnetic switch to detect the garage door open/close status. 

Can you please help me with Athom integration, and how we can integrate this?

Ps: I have the same HW setups with GPIO14 for Sonoff SV and Sonoff 1Chnl Smart Switch ready to test.

Thanks,

**Comment by robertklep** [comment]
Sensors aren't supported yet by the Sonoff app on Homey.

As an alternative, it might be possible to use the [MQTT Client](https://apps.athom.com/app/nl.scanno.mqtt) app to act on MQTT messages sent by the device, but it would require some MQTT skills.

**Comment by senadaruc** [comment]
Hmm thanks for the prompt response, my hope was that we can have 1 device for Open/Close and Sensor same time. 

Thanks,

**Comment by robertklep** [comment]
I agree, at some point I want to do a rewrite of the Sonoff app where it would autodetect all sensors/switches/relays connected to a device, and "fold" them into one Homey device, but I haven't yet gotten around to that.

**Comment by senadaruc** [comment]
I have all kind of Sonoff devices so if you need info, help, and testing let me know :) I am around.

**Comment by LastKnight62** [comment]
I also have a lot of Sonoff devices....

**Comment by AloisKlingler** [comment]
senadaruc,

did you try anything else with the magnetic door sensor? I was wondering if you would like to share the config and the flashed firmware, or can you tell me which GPIO is triggered on the ESP with the magnetic switch?
I have bought the same sensor and would like to save some search work which you did already before. ;)

thanks!
BR
Alois

**Comment by senadaruc** [comment]
Guys, I am on PTO until 7.May I will share when I am back at home.
